src/components/Users/UserQualifications.tsx

```
@@ -0,0 +1,274 @@
+      switch (field) {
+        case "doctor_experience_commenced_on":
+          if (states.form.user_type === "Doctor") {
+            if (states.form.doctor_experience_commenced_on === undefined) {
+              errors[field] = t("field_required");
+            } else if (
+              !validateNumber(states.form.doctor_experience_commenced_on ?? "")
+            ) {
+              errors[field] = t("doctor_experience_number_error");
+            } else {
+              const years = Number(states.form.doctor_experience_commenced_on);
+              if (isNaN(years) || years < 0 || years > 100) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)